### DISCUSSION TITLE
Cannot refer to current user info in PNP search handlebars

### DISCUSSION THREAD

**Discussion by shirleysham** [microsoft-search/pnp-modern-search#2912] [Q&A] [answered]
Hi,

I have a PNP search results part, and I'm trying to display the current user email on the results web part.  I can see the user mail in the debug mode:

![image](https://user-images.githubusercontent.com/121912343/232437178-5e99cbbb-0276-486e-a9b9-482942924d1f.png)

and I'm trying to refer to it by @root.context.user.email in my custom template as below:

![image](https://user-images.githubusercontent.com/121912343/232437821-13cfb251-616a-4f1d-a4f4-7b4a83684082.png)

But unable to display the user email.  I'm using a Sharepoint list as the query template.  Please help!! Many thanks!!

**Comment by wobba** [comment, marked as answer]
Tried with ` {{@root.context.user.email}}` inside a template and workes just fine in v4.

**Comment by shirleysham** [comment]
Thank you! Please close the log thanks.
